monai/optimizers/lr_finder.py

```diff
@@ -221,7 +231,9 @@ def __init__(
         # Save the original state of the model and optimizer so they can be restored if
         # needed
         self.model_device = next(self.model.parameters()).device
-        self.state_cacher = StateCacher(memory_cache, cache_dir=cache_dir)
```

Comment:
I feel this is not related to the issue, it's for the model and optimizer, do we want to use other protocols?

Reply:
Hi @wyli ,

Thanks for your review and comments.
I think the pickle protocol would be useful if the `self.model.state_dict()` is bigger than 4GB?

Thanks.